.setup/bin/setup_sample_courses.py

```diff
@@ -494,6 +512,8 @@ def parse_args():
     parser.add_argument("--courses_path", default=os.path.join(SETUP_DATA_PATH, "courses"),
                         help="Path to the folder that contains .yml files to use for course creation. Defaults to "
                              "../data/courses")
+    parser.add_argument("--install_dir", type=str, default="/usr/local/submitty", help="install path of submitty")
+    parser.add_argument("--data_dir", type=str, default="/var/local/submitty", help="data path of submitty")
```

Comment:
Can use the values of the globals as the defaults so that way don't have two separate places to edit if they change.

```suggestion
    parser.add_argument("--install_dir", type=str, default=SUBMITTY_INSTALL_DIR, help="install path of submitty")
    parser.add_argument("--data_dir", type=str, default=SUBMITTY_DATA_DIR, help="data path of submitty")
```